### ISSUE TITLE
Taris Lower City Javyar's Cantina - Twi'lek males lack appearance variety

### ISSUE THREAD

**Issue by DarthParametric** [KOTORCommunityPatches/K1_Community_Patch#72]
* Black Vulkars vs Calo Nord cutscene. The two Twi'lek males are both green. Possibly switch one to a different colour.
* The Twi'lek males in the bounty office are all using the same green head.  Possibly switch some to a different colour.

**Comment by JCarter426** [comment]
Calo's victims seem to transform:
![swkotor 2019-05-14 14-48-13-60](https://user-images.githubusercontent.com/46898647/57729258-b9a5c700-7663-11e9-907b-51c253e50389.png)
![swkotor 2019-05-14 14-48-21-96](https://user-images.githubusercontent.com/46898647/57729267-c5918900-7663-11e9-9f0e-968e3a8a71f0.png)

The bounty hunter also looked a little odd in commoner clothing. Wpuld be better to keep the Twi'lek clothes I think, although that might be down to his equipment slot.

**Comment by DarthParametric** [comment]
That's weird. The two bounty hunters near Zax should bother be wearing armour.

Calo's orange attacker should be using the Twi'lek body, like the green guy, not commoner clothes. I guess the switching wasn't noticeable before because they were identical, but I guess that must be a script issue.


INI setup (with some revised notes)
```
;
;CHANGE APPEARANCE ID TO ADDED BLUE HEAD FOR TIER 3, 2DAMEMORY11
;
[tar03_bounhun031.utc]
!Destination=modules\tar_m03ae.mod
Appearance_Type=48
AddField0=gff_tar03_bounhun031_Equip_ItemList_1_0
[gff_tar03_bounhun031_Equip_ItemList_1_0]
FieldType=Struct
Path=Equip_ItemList
Label=
TypeId=2
AddField0=gff_tar03_bounhun031_EquippedRes_0
[gff_tar03_bounhun031_EquippedRes_0]
FieldType=ResRef
Label=EquippedRes
Value=g_a_class7004
;
;APP 450 IS UNUSED ORANGE TWI'LEK BODY TEXTURE WITH ORANGE HEAD. PROBABLY TIER 3
;
[tar03_bounhun033.utc]
!Destination=modules\tar_m03ae.mod
Appearance_Type=450
AddField0=gff_tar03_bounhun033_Equip_ItemList_1_0
[gff_tar03_bounhun033_Equip_ItemList_1_0]
FieldType=Struct
Path=Equip_ItemList
Label=
TypeId=2
AddField0=gff_tar03_bounhun033_EquippedRes_0
[gff_tar03_bounhun033_EquippedRes_0]
FieldType=ResRef
Label=EquippedRes
Value=g_a_class4005
;
;APP 450 IS UNUSED ORANGE TWI'LEK BODY TEXTURE WITH ORANGE HEAD. PROBABLY TIER 3
;
[tar03_taproom033.utc]
!Destination=modules\tar_m03ae.mod
Appearance_Type=450
;
;2DAMEMORY10 = Alien_Twilek_Male_Orange_TwiBody
;
[bp_dead3.utc]
!Destination=modules\tar_m03ae.mod
Appearance_Type=2DAMEMORY10
```

**Comment by ebmar** [comment]
They have different UTC for before and after the ambush cutscene, hence the transformation. By memory; for the _after_ ones are `bp_deadxx` or something - sorry can't be more specific at the time being.

**Comment by DarthParametric** [comment]
Yeah clearly I made a hash job of it. I'll have to go back and redo it properly.

**Comment by JCarter426** [comment]
Here's how the bounty hunters look:
![swkotor 2019-05-14 23-44-18-55](https://user-images.githubusercontent.com/46898647/57747196-6a32bb80-76a2-11e9-8c0d-ed615c115188.png)

**Comment by DarthParametric** [comment]
What the hell. Can you crack open the module and see what appearance IDs they ended up with and what their inventory looks like?

**Comment by JCarter426** [comment]
tar03_bounhun033 matches your INI code, appearance 450 and equipped with class 4 armor.
tar03_bounhun032 appears to be the original setup, appearance 39 (green) with no armor.

**Comment by DarthParametric** [comment]
tar03_bounhun031 should have been the other one. Wasn't the third bounty hunter a Rodian?

**Comment by JCarter426** [comment]
Ah, it was 31, I just mistyped.

**Comment by DarthParametric** [comment]
I see what the problem is. It's that I'm an idiot. I didn't give them feats to wear the armour, so the game is unequipping it and they end up reverting to their naked appearance.

**Comment by DarthParametric** [comment]
Going to revert the addition of the TSL blue male Twi'lek head and switch that bounty hunter back to vanilla green. The two added blue appearance.2da rows and associated heads.2da row weren't used for anything else, so their config data can be removed altogether.
